src/components/Project/RedeemTokenViaExchange.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { CSSProperties, useContext } from 'react'
+import { Tooltip } from 'antd'
+import AMMPrices from 'components/AMMPrices'
+import { ThemeContext } from 'contexts/themeContext'
+
+export const RedeemTokenViaExchange = ({
```

Comment:
**Suggestion:** Maybe for consistency with `<AMMPrices />`, we just go with `<RedeemAMMPrices />` ?

Reply:
Definitely one of the hardest things for me are naming conventions 😄 

pretty good suggestion!